appium/saucetestcase.py

```diff
@@ -29,8 +30,8 @@
 sauce = SauceClient(SAUCE_USERNAME, SAUCE_ACCESS_KEY)
 
 
-def on_platforms(platforms):
-    def decorator(base_class):
+def on_platforms(platforms: List) -> Any:
+    def decorator(base_class: Any) -> None:
```

Comment:
Can we have a more specific type instead of Any for a class argument?

Reply:
https://stackoverflow.com/questions/44664040/type-hints-with-user-defined-classes/44664064

Reply:
Yes, it's just temporal. But I'll check your link, thanks.